tests/test_spline_type_conversion.py

```diff
@@ -18,9 +18,13 @@ def convert_and_compare_samples(spline, conversion_list):
         assert type(converted).__qualname__.lower().startswith(convert_to)
 
 
-class SplineTypeConversionTest(c.unittest.TestCase):
+class SplineTypeConversionTest(c.SplineBasedTestCase):
     def setUp(self):
-        self.z, self.r, self.b, self.n = c.all2p2d()
+        splines = c.get_all_spline_typs_as_list()
+        self.b = splines[0]
+        self.n = splines[1]
+        self.z = splines[2]
+        self.r = splines[3]
```

Comment:
Can this be avoided? this is very `c`onfusing

Reply:
the problem is not the assignment, though I would prefer:
```
[self.b, self.n, self.z, self.r] =  c.get_all_spline_typs_as_list()
```
however, it's the names that are unclear. I'd rather have it called. bezier, nurbs, etc. ... but still. We should not use the setup routine as functions are prone to interfere! That is the whole reason this pr exists.